hadoop-plugin/src/main/groovy/com/linkedin/gradle/azkaban/AzkabanHelper.groovy

```diff
@@ -267,7 +270,23 @@ class AzkabanHelper {
           azkProject.azkabanUserName = input.toString();
         }
 
-        input = console.readLine("New Azkaban Zip task (run 'ligradle tasks' to find existing Zip tasks) [enter to accept '${azkProject.azkabanZipTask}']: ");
+        HadoopZipExtension hadoopZipExtension = project.extensions.getByName("hadoopZip");
+        Map<String, CopySpec> zipMap = hadoopZipExtension.getZipMap();
+
+        if (zipMap.isEmpty()) {
+          logger.lifecycle("No zips configured in the hadoopZip block");
+        } else {
+          logger.lifecycle("\nFound the following zips in the hadoopZip block:");
+          def counter = 1;
+          zipMap.each { String zipName, CopySpec copySpec ->
+            logger.lifecycle("${counter++}. ${zipName.replace("hadoopZip","").replace("azkaban","")} - Enter \"${zipName}HadoopZip\" to use this zip");
+          }
+          logger.lifecycle("(Or you can use the name of any other Gradle zip task)");
+        }
+
+        //waiting for the above logger statements to flush
+        sleep(500);
+        input = console.readLine(" > New Azkaban Zip task [enter to accept '${azkProject.azkabanZipTask}']: ");
```

Comment:
Again there is an interruption from build message. It would be nice to know if there is a way to delay the interruption from build. One other way to make everything look even, is to take input by appending with the build message itself, as you did in your previous commit using
`console.format("<message>").flush();`